modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -7,13 +7,21 @@
 [id="registry-configuring-registry-storage-swift-trust_{context}"]
 = Configuring the Image Registry Operator to trust Swift storage
 
-You must configure the Image Registry Operator to trust {rh-openstack-first} Swift storage.
+You can configure the Image Registry Operator to trust {rh-openstack-first} Swift storage. This is an optional procedure depending on how your clients trust storage certificates.
```

Comment:
This description still doesn't seem quite right to me.
It is not to let  Image Registry Operator to trust storage, according to https://bugzilla.redhat.com/show_bug.cgi?id=1810461#c11, it should be the clients
In addition, how we define the clients here?

Reply:
👆🏼  +1

Reply:
Clients here could be for example a developer laptop's, but also a build in the cluster (I think builds is what this was first documented for).